app/src/main/java/org/oppia/android/app/administratorcontrols/AdministratorControlsActivityPresenter.kt

```diff
@@ -37,7 +37,7 @@ class AdministratorControlsActivityPresenter @Inject constructor(
     if (previousFragment != null) {
       activity.supportFragmentManager.beginTransaction().remove(previousFragment).commitNow()
     }
-    activity.supportFragmentManager.beginTransaction().add(
+    activity.supportFragmentManager.beginTransaction().replace(
```

Comment:
why this change?

Reply:
Change reverted: https://github.com/oppia/oppia-android/pull/3185/commits/5be55859c43b45f6e243ff4a95773fdeeda62ad8